doc/TILESET.md

```diff
@@ -383,6 +383,24 @@ Weighted variations are also possible, and can be combined with rotation:
 }
 ```
 
+Variant selection is based on the graffiti's text, so the same text will always result in the same variant shown.
+
+<details>
+<summary><h5>More graffiti variants - Spoile warning!<h5></summary>
```

Comment:
Don't make it a spoiler, this is technical documentation, not a thing for players.

Reply:
Removed tags and changed title